pkg/issuer/vault/setup.go

```diff
@@ -0,0 +1,21 @@
+package vault
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha1"
+	"k8s.io/api/core/v1"
+)
+
+const (
+	successVaultVerified = "VaultVerified"
```

Comment:
Not for you to fix as part of this PR: we should probably move these Reason messages into a generic package